pocld/daemon.hh

```diff
@@ -114,6 +125,8 @@ public:
 
   /* returns nullptr on error */
   VirtualContextBase *performSessionSetup(int fd, Request *R);
+  /* whether to use vsock for communication */
+  bool UseVsock;
```

Comment:
Shouldn't this be in the private section?

Reply:
vsock can not support TCP_NODELAY, so need a public flag , seems no need to add set/get function